File: 01. Iniciante/Java/URI1007.java
import java.io.IOException;
import java.util.Scanner;

// Alterar a classe para Main no URI
public class URI1007 {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        Integer A = sc.nextInt();
        Integer B = sc.nextInt();
        Integer C = sc.nextInt();
        Integer D = sc.nextInt();
        Integer DIF = A * B - C * D;
        System.out.printf("DIFERENCA = %d\n", DIF);
        sc.close();
    }
}

File: 01. Iniciante/C/URI1007.c
// URI 1007. Diferença

#include <stdio.h>

int main()
{
    int A, B, C, D, DIFERENCA;

    scanf("%d%d%d%d", &A, &B, &C, &D);

    DIFERENCA = (A * B - C * D);

    printf("DIFERENCA = %d\n", DIFERENCA);

    return 0;
}